src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -31,6 +31,9 @@
             + "Please note that record to the nearest KG/L is sufficient for inventory keeping.";
     public static final int MAXIMUM_AMOUNT = 999;
     public static final int MINIMUM_AMOUNT = 0;
+    public static final int MAXIMUM_LENGTH_OF_TAG = 50;
+    public static final String MESSAGE_EXCEED_MAXIMUM_TAG = "The length of tag entered is greater than 50 characters.\n"
+            + "Please re-enter tags which are not beyond 50 characters!";
```

Comment:
Instead of using double negatives, this could be better phrased as " .., tags of length below 50 characters"

Reply:
thanks! change it now